index.html

```diff
@@ -0,0 +1,21 @@
+---
+dateModified: "2025-02-12"
+language: "en"
+title: "OAG - template"
+layout: core
+pageclass: "oag-theme"
+noFooterMain: true
+breadcrumbs: false
+---
+
+<div class="container">	
+	<div class="row wb-eqht-grd">	
+		<h2 class="col-xs-12 h4  text-center mb-4 mt-5">Office of the Auditor General of Canada template</h2>
+		<div class="col-xs-12 col-sm-6 text-sm-right">
+			<p><a href="home.html" class="btn btn-link">Home page &rarr;</a></p>
```

Comment:
Is the "arrow" are a style widely used by them? like the default for any anchor? I recommend to use plain style here as like any other content page.